raiden/network/transport/matrix/utils.py

```diff
@@ -163,6 +163,22 @@ def force_user_presence(self, user: User, presence: UserPresence):
         """
         self._userid_to_presence[user.user_id] = presence
 
+    def populate_userids_for_address(self, address: Address, force: bool = False):
```

Comment:
I'd like to start adding `-> None` to such methods, to make more aggressive mypy options possible in the future.